### ISSUE TITLE
Feature Request: relative changing of window_gap and border_width with ±

### ISSUE THREAD

**Issue by zzggbb** [baskerville/bspwm#278]
To relatively change the window_gap or border_width, the following must be used:
    bspc config -d focused window_gap $(( `bspc config -d focused window_gap` + 3))

I think that window_gap and border_width should accept ± prefix to indicate relative resizing, so that the above can be accomplished with:
    bspc config window_gap +3

This would of course mean that negative window gaps can't be directly set, but I think this is far more useful (and `window -e` already supports this behaviour)

**Comment by netzverweigerer** [comment]
I agree with this, I've written a small wrapper script for this which implements this, but bspc seriously should advance in terms of usability here.


#!/bin/bash
# increase/decrease window gap in bspwm
current="$(bspc config -d focused window_gap)"
if [[ "$1" == "increase" ]]; then
  new=$((current+2))
	echo "$new"
	bspc config -d focused window_gap "$new"
fi
if [[ "$1" == "decrease" ]]; then
  new=$((current-2))
	bspc config -d focused window_gap "$new"
fi


All the best,

Armin

**Comment by neeasade** [comment]
you could use something like the following to do this transparently/without a custom script above. 

``` sh
bspc()
{
    _bspc=`which bspc`
    # window_gap will only ever be the 2nd arg
    if [ "$2" = "window_gap" ] && echo "$3" | grep -q "[+-]"; then
        $_bspc config window_gap $(( `$_bspc config window_gap` $3 ))
    else
        $_bspc $@
    fi
}
```
